spec/brcobranca/boleto/banco_bradesco_spec.rb

```diff
@@ -80,14 +78,14 @@
   end
 
   it 'Montar código de barras para carteira número 03' do
-    @valid_attributes[:valor] = 135.00
-    @valid_attributes[:dias_vencimento] = 1
-    @valid_attributes[:data_documento] = Date.parse('2008-02-01')
-    @valid_attributes[:numero_documento] = '777700168'
-    @valid_attributes[:conta_corrente] = '61900'
-    @valid_attributes[:agencia] = '4042'
-    @valid_attributes[:carteira] = '03'
-    boleto_novo = described_class.new(@valid_attributes)
+    valid_attributes[:valor] = 135.00
+    valid_attributes[:dias_vencimento] = 1
+    valid_attributes[:data_documento] = Date.parse('2008-02-01')
+    valid_attributes[:numero_documento] = '777700168'
+    valid_attributes[:conta_corrente] = '61900'
+    valid_attributes[:agencia] = '4042'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.